department-of-veterans-affairs/va.gov-cms
# VACMS-3487:  remove system fields step 2 of 3

## Description
See #3487 
Step 2 of 3. (Step 1 = remove fields from vets-web, Step 3 = remove fields from db in update hook)
Do not merge until vets-web pr https://github.com/department-of-veterans-affairs/vets-website/pull/15019 to remove fields from graphql is merged

## Testing done
VIsual / Behat (will need to update tests before changing from draft to ready for review)

## Screenshots
![disabled_fields](https://user-images.githubusercontent.com/2404547/99669772-92d90580-2a3d-11eb-90a3-45e3656efb5f.png)
![removed_fields](https://user-images.githubusercontent.com/2404547/99669773-93719c00-2a3d-11eb-8df7-adf545be661d.png)

## QA steps
- As as admin, go to `admin/structure/types/manage/health_care_region_page/form-display` and visually verify that `Regional Health Service Offerings.` is disabled
- As an admin, go to `admin/structure/types/manage/health_care_region_page/fields` and visually verify that Featured content and Press release blurbs fields have been removed.

### Definition of Done


### Select Team for PR review

### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
